kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/CreateQuizSection.vue

```diff
@@ -622,20 +497,7 @@
           params: { sectionIndex: this.allSections.length - 1 },
         });
       },
-      handleDragStart() {
-        // Used to mitigate the issue of text being selected while dragging
-        this.dragActive = true;
-      },
-      handleKeyboardDragDown(oldIndex, array) {
-        const newArray = this.moveDownOne(oldIndex, array);
-        this.handleQuestionOrderChange({ newArray });
-      },
-      handleKeyboardDragUp(oldIndex, array) {
-        const newArray = this.moveUpOne(oldIndex, array);
-        this.handleQuestionOrderChange({ newArray });
-      },
-      handleQuestionCheckboxChange(item, $event) {
-        $event.stopPropagation();
+      handleQuestionCheckboxChange(item) {
```

Comment:
Since this now only calls `this.toggleQuestionInSelection(item)` you could remove the `handleQuestionCheckboxChange` altogether and just pass that `toggleQuestionInSelection` directly as the prop.

Reply:
Thanks @nucleogenesis! Didnt noticed this.